_drafts/2021-01-08-ALTF4-20.md

```diff
@@ -0,0 +1,78 @@
+
+### Heading
+
+Translating takes time. We don't have that. Currently, articles come in at Thursday morning, we start translating at noon or in the afternoon, and have a first translation in the evening. But then someone wants to proofread it. And I can't remember the last time, we didn't have corrections after someone went through a translation. Corrections might not always be better though, even though they were well-meant[?] (we have a German word for that, "verschlimmbessern", *to worsen something by trying to improve it*); thus we now always have the original translator read through the corrections. But you guessed it, that takes time too. If we have a good day, and everyone is available, we are sometimes less than half an hour before release still busy ironing out some hiccups, correcting errors, and fiddling with non-synonyms.
+
+Sometimes these problems, especially the more technical ones, result in in changes for all languages, sometimes even english [maybe reference if touched upon]. For example quotes are where they are now, because we couldn't figure out how to incorporate translations while still keeping the original quote. [I feel like there should be something more, but I don't know what]
+
+### Heading
+
+I want to briefly talk about translating continuous text versus individual strings; in *TheEnemy42* touched upon [translating the game](https://alt-f4.blog/ALTF4-9/#a-story-about-translating-factorio-theenemy42) itself, and while keeping translations as close to the original as possible, while staying within a reasonable length, is an art form of itself, translating alt-f4 in a way that conveys all content, while still being enjoyable to read, is on a completely different level. Both suffer from a similar problem though, which *stringweasel* mentioned [in the very same issue](https://alt-f4.blog/ALTF4-9/#nauvis-archives-factorio-jargon-stringweasel): jargon. A large part of a games jargon is composed of contractions and abbreviations of in-game terms and specific already technical jargon from the real world, which often doesn't translate to other language, or even make sense in them.
+
+[still bad with transitions :/ ]
+
+With how much I've complained about, you might wonder, "Why do we even translate? Is it for the readers?". And while I do translate with hopes of reaching more people, the reality is, we don't:
+
+[graph, maybe a bar chart for readers]
+
+The reality is, at least I translate, because I enjoy it. It's sort of like programming, but instead of confusing flowcharts or messy thoughts in your head, we have an English text, and instead of translating to programming languages, we translate into real ones, which can be frustrating, as coding can be too, but all in all, is a fun puzzle.
+
+But you, dear reader, are one of those people we do reach, and if you want to help us out, it would be great of you could tell your local factorio community about alt-f4, be it translated or not!
```

Comment:
general remark eventually: it was written in last week issue that translation from translators was not the exact final step. Do we want to mention again the last technical review that is done on every translation work to ensure that it will work?